GVFS/GVFS.Service/GVFSMountProcessManager.cs

```diff
@@ -6,34 +6,34 @@
 
 namespace GVFS.Service
 {
-    public class GVFSMountProcess : IDisposable

```

Comment:
This name confused me to no-end! 🤦‍♂️  I thought this represented a mount process instance itself, but it's really a sort of factory for processes right? I originally added a `StopMount(string repoRoot)` method to this along with the rename which would be invoked on `VolumeUnavailable` but removed that logic in favour of open question 1.

Reply:
> I thought this represented a mount process instance itself, but it's really a sort of factory for processes right?

I'm not sure that I'd say it's a factory (as it doesn't produce any objects) but you have the right idea.  This class is a helper for calling the `gvfs` CLI.

Part of the naming confusion might be because prior to calling `gvfs mount` this class would start `GVFS.Mount.exe` directly.

Reply:
I would leave the name unchanged. I can understand the confusion, but it's analogous to the .NET `Process` class, or our own `GitProcess` class. The class is not the process - it's code for interacting with the process.